modules/mirror-registry-flags.adoc

```diff
@@ -12,6 +12,7 @@ The following flags are available for the _mirror registry for Red Hat OpenShift
 | Flags | Description
 | `--autoApprove` | A boolean value that disables interactive prompts. If set to `true`, the `quayRoot` directory is automatically deleted when uninstalling the mirror registry. Defaults to `false` if left unspecified.
 | `--initPassword` | The password of the init user created during Quay installation. Must be at least eight characters and contain no whitespace.
+|`--initUser string` | Shows the password of the initial user. Defaults to `init` if left unspecified. 
```

Comment:
Should be `The username of the initial user`, the original description is wrong in help page, DEV will fix it in next release